src/app/routes/index.js

```diff
@@ -1,10 +1,16 @@
-import pathOr from 'ramda/src/pathOr';
-import Article from '../containers/Article';
-import FrontPage from '../containers/FrontPage';
-import RadioPage from '../containers/RadioPage';
-import CpsAssetPage from '../containers/CpsAssetPage';
-import ErrorPage from '../containers/Error';
+import path from 'ramda/src/path';
 import getInitialData from './getInitialData';
+
+// Pages
+import Article from '../pages/Article';
+import FrontPage from '../pages/FrontPage';
+import RadioPage from '../pages/RadioPage';
+import CpsMap from '../pages/CpsMap';
+import CpsSty from '../pages/CpsSty';
+import CpsPgl from '../pages/CpsPgl';
```

Comment:
I know we can't do anything about CPS's cryptic 3-letter asset types at the lowest level, but we aren't obliged to  replicate them in our component naming, i.e. if these were called `CpsMediaAssetPage`, `CpsPhotoGallery` etc. might that be easier for newcomers to make sense of without sacrificing anything?

Reply:
very true, happy to make this change. Are you thinking the files names should be changed too or just the import names?

Reply:
As this is already in ready for test, I'll raise a new PR with naming changes and discuss it with the team.